src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java

```diff
@@ -98,7 +122,12 @@ void shouldReturnTemplateContentForJoint() {
             entry(DATE_OF_HEARING, "8 November 2021"),
             entry(TIME_OF_HEARING, "14:56 pm"),
             entry(BEFORE_DATE_OF_HEARING, "1 November 2021"),
-            entry(HAS_FINANCIAL_ORDERS, true)
+            entry(HAS_FINANCIAL_ORDERS, true),
+            entry(CTSC_CONTACT_DETAILS, buildCtscContactDetails()),
+            entry(DIVORCE_AND_DISSOLUTION_HEADER, "Divorce and Dissolution"),
```

Comment:
See related comment

Reply:
Removed magic strings and replaced with constants